README.md

```diff
@@ -1 +1,4 @@
-# CityElf
\ No newline at end of file
+# CityElf
```

Comment:
You should write how to start an app for other people in team:
- gradlew command
- start in IDE (tomcat port, where it is set up)
- also mention about intellij idea bug